config/symbols.us.stcen.txt

```diff
@@ -1,3 +1,5 @@
+g_ScrollDeltaX = 0x80097908;
```

Comment:
Stages start at 0x80180000, DRA starts at 0x800A0000, so this is actually a main symbol (starting from 0x80010000) it seems. Not sure but that might be your compile issue.

Reply:
This will help understanding the memory layout and where to put the symbols: https://github.com/Xeeynamo/sotn-decomp/wiki/Internals

Reply:
Thanks, I hadn't added the symbol to `symbols.hd.txt`.